src/formatter.test.ts

```diff
@@ -506,6 +506,48 @@ Yes!
     stable(expected);
   });
 
+  it('human-readable ordered lists', () => {
```

Comment:
```suggestion
  it('increment list numbers option', () => {
```

Reply:
updated it in a commit while you were typing!